5/ansible-5.2.0.deps

```diff
@@ -0,0 +1,101 @@
+_ansible_version: 5.2.0
+_ansible_base_version: 2.12.0
```

Comment:
Needs to be rebuilt after modifying this manually with 2.12.1.

What bug was this again ?